runtime/cere/src/lib.rs

```diff
@@ -1679,6 +1670,16 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_nomination_pools_runtime_api::NominationPoolsApi<
```

Comment:
What about this line ? Do we need the same for the `cere-dev`, or it can be safely skipped?

Reply:
Thanks. The line is added to cere-dev